cloudferrylib/os/compute/nova_compute.py

```diff
@@ -403,6 +370,23 @@ def _deploy_flavors(self, flavors, tenant_map):
                     self.add_flavor_access(_flavor['meta']['id'],
                                            tenant_map[tenant])
 
+    def _deploy_quotas(self, quotas, tenant_map, user_map=None):
+        for _quota in quotas:
+            old_tenant_id = _quota['tenant_id']
```

Comment:
Does this handle the case when old tenant ID does not exist anymore? E.g. in the following scenario:

1. Tenant `t1` is created
2. Quota `q1` is associated with a tenant `t1`
3. Tenant `t1` is removed
4. Quota `q1` is still associated with non-existing tenant `t1`, will `_quota['tenant_id']` have any tenant ID? Will it fail with `KeyError`?